src/site/content/en/blog/tabbed-application-mode/index.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: Tabbed application mode for PWAs
+subhead: Work on more than one document at a time with tabs in your Progressive Web App
+authors:
+  - thomassteiner
+date: 2021-02-16
+description: |
+  Tabbed application mode allows Progressive Web App developers to add a tabbed document interface
+  to their standalone PWAs.
+hero: hero.jpg
+alt: Dictionary thumb index.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - progressive-web-apps
+---
+
+{% Aside %}
+  Tabbed application mode is part of the
+  [capabilities project](https://web.dev/fugu-status/) and is currently in development. This post
+  will be updated as the implementation progresses. Tabbed application mode is an early-stage
+  exploration of the Chrome team. It is not ready for production yet.
+{% endAside %}
+
+In the world of computing, the [desktop metaphor](https://en.wikipedia.org/wiki/Desktop_metaphor) is
+an interface metaphor which is a set of unifying concepts used by graphical user interfaces (GUI) to
+help users interact more easily with the computer. In keeping with the desktop metaphor, GUI tabs
+are modeled after traditional card tabs inserted in books, paper files, or card indexes. A _tabbed
+document interface_ (TDI) or tab is a graphical control element that allows multiple documents or
+panels to be contained within a single window, using tabs as a navigational widget for switching
+between sets of documents.
+
+Progressive Web Apps can run in [various display modes](/add-manifest/#display) determined by the
+`display` property in the Web App Manifest. Examples are `fullscreen`, `standalone`, `minimal-ui`,
+and `browser`. These display modes follow a
+[well-defined fallback chain](https://w3c.github.io/manifest/#dfn-fallback-display-mode)
+(`"fullscreen"` → `"standalone"` → `"minimal-ui"` → `"browser"`). If a browser does not support a
+given mode, it falls back to the next display mode in the chain.
+
+## What is tabbed application mode
+
+Something that has been missing from the platform so far is a way to let PWA developers offer their
+users a tabbed document interface, for example, to enable editing different files in the same PWA
+window. Tabbed application mode closes this gap.
+
+{% Aside %}
+  This feature is about having a standalone app window with multiple tabs (containing
+  separate documents inside the app scope) inside it. It is not to be confused with
+  `"display": "browser"`, which does already exist and has a separate meaning (specifically, that
+  the app is opened in a regular browser tab).
+{% endAside %}
+
+### Suggested use cases for tabbed application mode
+
+Examples of sites that may use tabbed application mode include:
+
+- Productivity apps that let the user edit more than one document (or file) at the same time.
+- Communication apps can let the user have conversations in different rooms per tab.
```

Comment:
```suggestion
- Communication apps that let the user have conversations in different rooms per tab.
```

Reply:
Somewhere there's some writing advice about mixing up wording to keep a piece from being monotonous. In the case of lists, the rule to use parallel grammatical structures has priority. (Structures can be repeated for rhetorical effect too. See [the second paragraph of this](https://www.archives.gov/founding-docs/declaration-transcript).

Reply:
That link predates most style guides and modern developments in language usage, so it seems to 'break' rules that I've been trying to share.